magma_ff/create_metawfr.py

```diff
@@ -6,28 +6,106 @@
 #
 ################################################
 
-################################################
-#   Libraries
-################################################
 import datetime
 import json
 import uuid
+from typing import List
 
 from dcicutils import ff_utils
 
-# magma
 from magma_ff.metawfl import MetaWorkflow
 from magma_ff.metawflrun import MetaWorkflowRun
-from magma_ff.utils import make_embed_request
+from magma_ff.utils import JsonObject, keep_last_item, make_embed_request
 
 
 FILE_FORMAT = "file_format"
+SAMPLE_PROCESSING_TYPE = "SampleProcessing"
+SAMPLE_TYPE = "Sample"
+TYPE = "@type"
 UUID = "uuid"
 
 
-################################################
-#   MetaWorkflowRunCreationError
-################################################
+def create_meta_workflow_run(
+    item_identifier: str,
+    meta_workflow_identifier: str,
+    auth_key: JsonObject,
+    post: bool = True,
+    patch: bool = True,
+) -> None:
```

Comment:
could you change this to return the generated object dict? I need to be able to access its generated uuid once created to save on the MetaWorkflowRun Handler. Otherwise keep everything else the same, the automated posting and patching is quite useful!